packages/vue-app/template/client.js

```diff
@@ -128,7 +128,7 @@ function mapTransitions(Components, to, from) {
 
 async function loadAsyncComponents(to, from, next) {
   // Check if route path changed (this._pathChanged), only if the page is not an error (for validate())
-  this._pathChanged = !!app.nuxt.err || from.path !== to.path
+  this._pathChanged = Boolean(app.nuxt.err) || from.path !== to.path
```

Comment:
neat: If we swipe conditions, we can prevent unnecessary Boolean cast where  `from.path !== to.path`

```suggestion
  this._pathChanged = from.path !== to.path || Boolean(app.nuxt.err)
```

Reply:
Do you mean this? :relaxed:
 
```suggestion
  this._pathChanged = from.path !== to.path || app.nuxt.err
```

Reply:
Where actually it depends. `|| app.nuxt.err` can cause `_pathChanged` to be an error instead of instance of `err`. This is why we use `!!` or `Boolean`. If such type safety is not important we can omit `Boolean` everywhere :)